Production/plugins/SummaryProducer.cc

```diff
@@ -19,8 +19,8 @@ This file is part of https://github.com/hh-italian-group/h-tautau. */
 #include "h-tautau/Core/include/TauIdResults.h"
 #include "h-tautau/Core/include/TriggerResults.h"
 #include "h-tautau/Production/interface/GenTruthTools.h"
-#include "h-tautau/Production/interface/TriggerFileDescriptor.h"
-#include "h-tautau/Production/interface/TriggerFileConfigEntryReader.h"
+#include "h-tautau/Core/include/TriggerFileDescriptor.h"
+#include "h-tautau/Core/include/TriggerFileConfigEntryReader.h"
```

Comment:
now Summary shouldn't have any trigger-related information, right? remove the includes.